src/main/java/controller/LottoGameController.java

```diff
@@ -0,0 +1,68 @@
+package controller;
+
+import java.util.List;
+
+import domain.EarningRate;
+import domain.Lotto;
+import domain.LottoBall;
+import domain.LottoGenerator;
+import domain.LottoMoney;
+import domain.Lottos;
+import domain.WinningLotto;
+import domain.WinningResult;
+import view.LottoGameView;
+
+public class LottoGameController {
+	private final LottoGameView lottoGameView;
+	private final LottoGenerator lottoGenerator;
+
+	public LottoGameController(LottoGameView lottoGameView, LottoGenerator lottoGenerator) {
+		this.lottoGameView = lottoGameView;
+		this.lottoGenerator = lottoGenerator;
+	}
+
+	public void play() {
+		LottoMoney lottoMoney = getLottoMoney();
+		Lottos lottos = lottoGenerator.generateLottos(lottoMoney.calculateLottoCount());
+
+		lottoGameView.printPurchasedLottos(lottos);
+		Lotto winningLottoNumbers = getWinningLottoNumbers();
+		WinningLotto winningLotto = getWinningLotto(winningLottoNumbers);
+		WinningResult winningResult = WinningResult.of(winningLotto, lottos);
+
+		lottoGameView.printWinningRank(winningResult);
+		lottoGameView.printEarningRate(EarningRate.of(lottoMoney, winningResult.getWinningMoney()));
+	}
+
+	private LottoMoney getLottoMoney() {
+		try {
+			return new LottoMoney(lottoGameView.getLottoMoneyInput());
+		} catch (IllegalArgumentException e) {
+			System.out.println("[ERROR] " + e.getMessage());
+		}
+		return getLottoMoney();
```

Comment:
catch 문 안에서 다시 `getLottoMoney` 객체를 호출하도록 변경해주면 될 것 같아요. 😃

```suggestion
                     return getLottoMoney();
		}
```